frontend/Container.js

```diff
@@ -37,14 +43,42 @@ class Container extends React.Component {
     },
     extraTabs: {[key: string]: () => React$Element},
   };
+  state: State;
+
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      isVertical: (window.innerWidth < 500),
```

Comment:
Let's extract a constant since it's used in two places and we might forget one without the other by mistake.
Alternatively you can make a `computeState(window)` function and use it in both places.